huggingface/pytorch/inference/docker/1.9/py3/sdk1.16.1/Dockerfile.neuron

```diff
@@ -7,7 +7,7 @@ LABEL dlc_major_version="1"
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
-ARG MMS_VERSION=1.1.5
+ARG TS_VERSION=0.4.0
```

Comment:
Ehm why did you add torchserve? We are not using Torchserve for Inference? The Inference Toolkit is built on top of MMS for both frameworks. https://github.com/aws/sagemaker-huggingface-inference-toolkit

Reply:
oh.. Will add mms back. Thanks.

Reply:
Can you set MMS to 1.1.7 when reverting?